src/Symfony/Component/Process/Process.php

```diff
@@ -303,6 +303,10 @@ public function start(callable $callback = null, array $env = array())
             $ptsWorkaround = fopen(__FILE__, 'r');
         }
 
+        if (!file_exists($this->cwd)) {
+            @trigger_error('The provided cwd does not exist. Command is currently ran against getcwd()', E_USER_DEPRECATED);
```

Comment:
oh, and the message should tell since which version this deprecation exist (see other deprecation messages in the code)